doc/nrf/config_and_build/configuring_app/output_build_files.rst

```diff
@@ -28,129 +28,94 @@ Common output build files
+| :file:`zephyr.hex`                   | Default full image (name configurable with :kconfig:option:`CONFIG_KERNEL_BIN_NAME`).                  | * Programming board targets with :ref:`NSPE <app_boards_spe_nspe>` or single-image. |
+|                                      | In a project with multiple images, several :file:`zephyr.hex` files are generated, one for each image. | * Testing DFU procedure with nrfjprog (programming directly to device).             |
++--------------------------------------+--------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------+
+| :file:`merged.hex`                   | The result of merging all HEX files for all images for a specific core                                 | * Programming multi-core application.                                               |
+|                                      | In multi-core builds, several :file:`merged_<core>.hex` fields                                         | * Testing DFU procedure with nrfjprog (programming directly to device).             |
+|                                      | are generated, where *<core>* indicates the CPU core.                                                  |                                                                                     |
++--------------------------------------+--------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------+
+| :file:`tfm_s.hex`                    | Secure firmware image created by the TF-M build system in the background of the Zephyr build.          | Programming :ref:`SPE-only <app_boards_spe_nspe>` and multi-core board targets.     |
+|                                      | It is used together with the :file:`zephyr.hex` file, which is intended for the Non-Secure             |                                                                                     |
+|                                      | Processing Environment (NSPE). Located in :file:`build/tfm/bin`.                                       |                                                                                     |
++--------------------------------------+--------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------+
+| :file:`zephyr.signed.bin`            | Image update file used to create :file:`dfu_application.zip` for multi-core DFU.                       | DFU process for single or multi-core board targets                                  |
```

Comment:
I'm not sure if this is mentioned anywhere (at least I couldn't find it). Could we describe that this file is not located in the main build directory, like e.g. `merged.hex`, but you have to go to the `build/<your_application_name>/zephyr` to find it? The same for the `zephyr.signed.hex` below.

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/15845/files#diff-b701f12593e0a843cebbda466d8884f3f0f7b88d3344ee1f9751e8c4fa178263R354 ?